packages/sources/mycryptoapi/README.md

```diff
@@ -0,0 +1,30 @@
+# Chainlink External Adapter for MyCryptoApi
+
+### Input Parameters
+
+| Required? |   Name   |     Description     |            Options             | Defaults to |
+| :-------: | :------: | :-----------------: | :----------------------------: | :---------: |
+|           | endpoint | The endpoint to use | [gasprice](#gasprice-Endpoint) | `gasprice`  |
+
+---
+
+## Gas Price Endpoint
+
+### Input Params
+
+| Required? |  Name   |    Description    |               Options                | Defaults to |
+| :-------: | :-----: | :---------------: | :----------------------------------: | :---------: |
+|    🟡     | `speed` | The desired speed | `safeLow`,`standard`,`fast`,`fastest` |  `fast`  |
+
+### Output Format
+
```

Comment:
@boxhock should we display the verbose output instead here?

Reply:
Agreed, that could be useful

Reply:
the verbose output? Can you explain this to me a little bit? please

Reply:
done